counsel.el

```diff
@@ -1695,6 +1695,14 @@ Skip some dotfiles unless `ivy-text' requires them."
 (defvar counsel-find-file-speedup-remote t
   "Speed up opening remote files by disabling `find-file-hook' for them.")
 
+(defun preselect-file ()
```

Comment:
This should probably be called `counsel-preselect-file`

Reply:
Oh, right, fixing.